doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -25,15 +25,131 @@
 Traceability to Reference Model.
 
 <a name="2.3"></a>
-## 2.3 Requirements.
+## 2.3 Containerised Platform Requirements.
+
+The Reference Model (RM) defines the NFVI to consist of the physical resources, virtualised resources and a software managment system.  In the virtualised world, the NFVI Software Management System (Virtualised Infrastructure Manager -- VIM) consists of the Guest Operating System, Hypervisor and, if needed, other software such as libvirt.  And the VIM is responsible for, among others, tenant management, resources management, inventory, scheduling, and access management. Now consider, the containerisation equivalent.
+
+Containerised Infrastructure Manager (CIM): a functional block that is responsible for controlling and managing the containerised NFVI compute, storage and network resources. Note CIM supports the Life Cycle Management of conatinerised and the underlying physical resources.
```

Comment:
I think here we should list requirements from the RM not definitions from ETSI NFV.

Reply:
I worry about the term "Containerised Infrastructure Manager" because, as I've raised before and as per the definitions we've taken from kubernetes.io, a container is a packaged bit of software with all dependencies. The container orchestration/scheduler is about ensure that bit of software is running, and it's the container runtime that manages the infrastructure resources for that container.  The container isn't an infrastructure "thing" and so I don't think this description is accurate.

Reply:
I would prefer to leave this line out here.  We have some terminology definitions in chapter 1, and those will get described further in chapters 3 and 4 so I don't think this is needed here.

Reply:
@tomkivlin Can you open a PR for Ch 1 to include this definition. Deleting from Chapter 2

Reply:
@pgoyal01 I think the existing definitions for Kubernetes Cluster are sufficient.  Resolving this conversation as you've removed the paragraph from this PR.